src/convenience/session.ts

```diff
@@ -108,7 +108,7 @@ export interface StorageAdapter<T> {
 /**
  * Options for session middleware.
  */
-export interface SessionOptions<S> {
+export interface SessionOptions<S, C extends Context> {
```

Comment:
```suggestion
export interface SessionOptions<S, C extends Context = Context> {
```
or else it will be breaking

Reply:
Are you able to produce a breaking example? I tried to do only
```ts
const sus = session();
```

But it seems to assume the extending class as the implied default type parameter.

Reply:
Hmm.. I guess it could be breaking if somebody (for whatever reasons) was trying to do:
```ts
const opts: SessionOptions<SessionFlavor<{}>> = {}
```

Now, they would have to change it to
```ts
const opts: SessionOptions<SessionFlavor<{}>, MyContext> = {}
```

A rather unlikely case, but I guess..